### ISSUE TITLE
quell “InfluxDbReporter report is called” ?

### ISSUE THREAD

**Issue by srl295** [networknt/light-4j#53]
[this doc](https://networknt.github.io/light-java/tutorials/ms_chain/) says:
> When you start a service, you might realized that every few minutes, “InfluxDbReporter report is called” will be shown up on the console. By default, all services will try to report metrics info to Influxdb and subsequently viewed from Grafana dashboard.

Is there a way to get `InfluxDbReporter report is called` to _not_ show up on the console?

**Comment by stevehu** [comment]
This has been removed in the latest code in master. Waiting for the next release 1.2.8. Thanks for pointing this out.

**Comment by stevehu** [comment]
@srl295 I have released 1.2.8 yesterday and this console log will be gone. Could you please review and let me know if this is resolve or not. Thanks.

**Comment by stevehu** [comment]
with up coming 1.3.2 release, this has been changed to a debug logging statement to assist developers. 

```
        if(logger.isDebugEnabled()) logger.debug("InfluxDbReporter report is called with counter size " + counters.size());
```
